clients/hadoopfs/src/test/java/io/lakefs/S3FSTestBase.java

```diff
@@ -33,7 +33,7 @@
  * Base for all LakeFSFilesystem tests that need to access S3.  It adds a
  * MinIO container to FSTestBase, and configures to use it.
  */
-abstract class S3FSTestBase extends FSTestBase {
+public abstract class S3FSTestBase extends FSTestBase {
```

Comment:
Please document where this class is being used. This is easily revertible and breaks the usage you have in mind here